### PR TITLE
Update dependency versions, remove usage of dep, update maxctrl_expor…

### PR DESCRIPTION
**Pull request by Vetal1977**
- Updated dependency versions
- Removed usage of dep
- Updated maxctrl_exporter version
- Added `Contribution` section into README

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,11 +3,11 @@ module github.com/maxctrl_exporter
 go 1.15
```
Comment (pgporada) [inline go.mod:3]: Bump this go to 1.17

**Review by pgporada — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (Vetal1977) [reply go.mod]: @pgporada Updated. Thank you!

**Comment by ktugan** [comment]
Sorry @Vetal1977 I saw this one too late. I see you pushed already to master. Closing this PR.

**Comment by ktugan** [comment]
My bad. I see that the force push was on the branch itself :) Reopening.

**Pull request closed without merging**